src/main/kotlin/com/example/tlitter/controllers/UserController.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.tlitter.controllers
+
+import io.swagger.annotations.ApiOperation
+import io.swagger.annotations.ApiResponse
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/users")
+class UserController {
+    @PostMapping("login")
+    @ApiOperation(value = "認証を行いJWTを返す")
+    @ApiResponse(code = 404, message = "不正なEmailまたはPasswordです")
+    fun login(@RequestBody loginDTO: LoginDTO): JwtDTO {
+        return JwtDTO("This is JWT!")
+    }
+
+    class LoginDTO(
```

Comment:
[imo]

あまり詳しくないのですがI/Fのモデル名にはDTOってつけないような気もします
```suggestion
    class LoginRequest(
```
とかでどうでしょ？

Reply:
[imo]

モデル定義はcontrollerの責務外だと思うので`tlitter.request.user`みたいな感じでパッケージ切っていただければと思います(パッケージ名をmodelにしてしまうと後でDBにわたすモデルの置き場の命名に困りそうな気がしています)

Reply:
こちらで対応しました！ f8d3957